src/renderer/reader/components/ReaderSettings.tsx

```diff
@@ -390,63 +390,65 @@ interface ITable {
 
 const Slider = ({ value, option, set }: { value: string, option: ITable, set: (a: Pick<ReaderConfig, "pageMargins" | "wordSpacing" | "letterSpacing" | "paraSpacing" | "lineHeight">) => void }) => {
     const [currentSliderValue, setCurrentSliderValue] = React.useState(option.defaultValue);
+    const [currentIndex, setCurrentIndex] = React.useState(() => optionsValues[option.parameter].findIndex((el) => el === option.defaultValue));
 
     React.useEffect(() => {
-        setCurrentSliderValue(value.replace(/rem/g, ""));
-    }, [value]);
+        setCurrentSliderValue(value);
+        const newIndex = optionsValues[option.parameter].findIndex((el) => el === value);
```

Comment:
same remark as above regarding sanity-checking `optionsValues[option.parameter]` for `undefined`.

Reply:
Changes done